Makefile

```diff
@@ -160,7 +160,7 @@ ENVTEST ?= $(LOCALBIN)/setup-envtest
 
 ## Tool Versions
 KUSTOMIZE_VERSION ?= v3.8.7
-CONTROLLER_TOOLS_VERSION ?= v0.9.2
+CONTROLLER_TOOLS_VERSION ?= v0.10.0
```

Comment:
Please don't change this version. This is closely tied to the sdk version and might have unintended consequence. We can update this once we upgrade the sdk version.

Reply:
OK I can drop this.

Reply:
> Please don't change this version. This is closely tied to the sdk version and might have unintended consequence. We can update this once we upgrade the sdk version.

Which SDK do you mean, is it Operator SDK mentioned [here](https://github.com/confidential-containers/operator/blob/main/docs/DEVELOPMENT.md)? I just want to say that currently in our local setup, the controller side golang code use Kubebuilder annotation to auto-generate stuffs, it looks like we do not have Operator SDK installed but the operator image build is successful.